lib/okta.go

```diff
@@ -91,13 +121,14 @@ func NewOktaClient(creds OktaCreds, oktaAwsSAMLUrl string, sessionCookie string,
 	}
 
 	return &OktaClient{
-		Organization:   creds.Organization,
-		Username:       creds.Username,
-		Password:       creds.Password,
-		OktaAwsSAMLUrl: oktaAwsSAMLUrl,
-		CookieJar:      jar,
-		BaseURL:        base,
-		MFADevice:      mfaDevice,
+		Organization:   	creds.Organization,
+		Username:       	creds.Username,
+		Password:       	creds.Password,
+		OktaAwsSAMLUrl: 	oktaAwsSAMLUrl,
+		CookieJar:      	jar,
```

Comment:
Above, we should add a comment on `Organization` saying we're setting it just for backwards compatibility.